doc/index.rst

```diff
@@ -1,10 +1,71 @@
-The Plutus Platform
+Plutus Tools SDK User Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
-To read more about the Platform, see :ref:`what_is_the_plutus_platform`.
-To get started using the Platform see :ref:`plutus_getting_started`.
+Intended Audience
+-------------------------------
 
+This user guide is intended for developers who are authoring distributed applications 
+("DApps") by using smart contracts on the Cardano blockchain. 
+
+.. note::
+   If you are a developer who wants to contribute to the Plutus Tools SDK project, 
+   please refer to documentation residing in the `Plutus Tools SDK repository <https://github.com/
+   input-output-hk/plutus-apps>`_.
+
+Plutus Tools SDK
+===============================
+
+The Plutus Tools SDK is a collection of off-chain infrastructure resources built 
+for external developers. The `Plutus Tools SDK repository <https://github.com/
+input-output-hk/plutus-apps>`_ supports the underlying resources that developers 
+need who are writing full applications using Plutus in Haskell, including off-chain 
+code. The term “off-chain code” refers to the part of a contract application’s code 
+which runs outside of the blockchain. Off-chain code responds to events happening 
```

Comment:
Not entirely sure but wasn't it the *on-chain* code that responded to events from off-chain? At least that's what I vaguely remember from the Plutus Pioneer Program.

Reply:
The onchain checks whether certain conditions are met with limited scope. The only "response" it can perform is to invalidate a transaction by returning an error. Off-chain code can respond to any event happening, including on this blockchain or from any other interface.